src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'

```

Comment:
this url is part of a local env variable

Reply:
Got it -- we arent using this anymore bc the endpoints are built into amplify when using dataStore.